AirCasting/SessionViews/SingleMeasurementViewModel.swift

```diff
@@ -0,0 +1,21 @@
+// Created by Lunar on 26/01/2022.
+//
+
+import Foundation
+import Combine
+
+class SingleMeasurementViewModel: ObservableObject {
+    let settings: UserSettings
```

Comment:
well, this is not a correct approach to create viewmodel and still expose the original dependency thru it. I see 2 options here:
1) make this `UserSettings` dependency property private and expose some proper primitive value. Best would be to move the whole value here:
```Swift
@Published var measurementValue: Double
```
this way we can kill two birds with one stone - move some non-view logic from the `SingleMeasurementView` and hide this dependency.

2) Don't introduce this viewmodel just yet and instead properly implement `ObjervableObject` for `UserSettings` (currently this conformance is there only to be able to inject it via `@EnvironmentObject`, we can make it properly observable by calling `objectWillChange.send()` in each of the setters). This method would be easier to implement for sure, but will introduce some more tech debit instead of cleaning things up.

your call :)

Reply:
Okay, I'm choosing the second one, thanks 👌🏻